library/driver/src/nativeMain/kotlin/io/toxicity/sqlite/mc/driver/PlatformDriver.kt

```diff
@@ -262,3 +280,28 @@ private fun SqlSchema<QueryResult.Value<Unit>>.upgrade(
         migrate(it, oldVersion.toLong(), newVersion.toLong(), callbacks = afterVersions)
     }
 }
+
+@Suppress("NOTHING_TO_INLINE")
+private inline fun MutableMap<String, String>.journalMode(): JournalMode {
+    return remove("journal_mode")?.let { JournalMode.forString(it) } ?: JournalMode.DELETE
+}
+
+@Suppress("NOTHING_TO_INLINE")
+private inline fun MutableMap<String, String>.busyTimeout(): Int {
+    return remove("busy_timeout")?.toIntOrNull() ?: 5_000
+}
+
+@Suppress("NOTHING_TO_INLINE")
+private inline fun MutableMap<String, String>.pageSize(): Int? {
+    return remove("page_size")?.toIntOrNull()
+}
+
+@Suppress("NOTHING_TO_INLINE")
+private inline fun MutableMap<String, String>.foreignKeys(): Boolean {
+    return remove("foreign_keys")?.toIntOrNull()?.let { it == 1 } ?: false
+}
+
+@Suppress("NOTHING_TO_INLINE")
+private inline fun MutableMap<String, String>.recursiveTriggers(): Boolean {
+    return remove("recursive_triggers")?.toIntOrNull()?.let { it == 1 } ?: false
+}
```

Comment:
`SQLite` also accepts true/false, not just 0/1. If "false" or "true" is passed, this should also be accounted for.

Reply:
Resolved in commit bf8bc40